sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -654,6 +658,40 @@ def _update_check_handler(
             full_summary=summary,
         )
 
+    def update_linter_check(
+        self,
+        status: GithubCheckStatus,
+        conclusion: t.Optional[GithubCheckConclusion] = None,
+    ) -> None:
+        if not self._context.config.linter.enabled:
+            # Linter is not enabled, skip check
+            return
+
+        def conclusion_handler(
+            conclusion: GithubCheckConclusion,
+        ) -> t.Tuple[GithubCheckConclusion, str, t.Optional[str]]:
+            test_summary = (
```

Comment:
Is `test_summary` the right name for this? Seems like it could've been copied from a unit testing handler?